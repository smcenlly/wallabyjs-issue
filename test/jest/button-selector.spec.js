/* eslint-disable */
/**
 * @jest-environment jsdom
 */

import { mount, createLocalVue } from '@vue/test-utils';
import Quasar, { QBtn, QCarousel } from 'quasar';
import { ButtonSelector as COMPONENT } from '@components/button-selector/index.js';


console.error = jest.fn(e => { throw new Error(e); });

describe('<button-selector>', () => {
  const localVue = createLocalVue();
  localVue.use(Quasar, { components: ['QBtn']});
  let wrapper, vm;

  beforeEach(() => {
    wrapper = mount(COMPONENT, {
        propsData: {
            name: 'circle',
        },
        localVue,
    });
    vm = wrapper.vm;
  });

  it('Loads successfully', () => {
    expect(wrapper.isVueInstance()).toBe(true);
  });

  it('Produces expected HTML', () => {
    expect(wrapper.vm.$el).toMatchSnapshot();
  });

});
