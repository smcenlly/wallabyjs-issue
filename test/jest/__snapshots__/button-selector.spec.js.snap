// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<button-selector> Produces expected HTML 1`] = `
<div
  class="row buttonSelector-container"
  name="circle"
/>
`;
